Sources/JobsPostgres/Migrations/CreateJobQueue.swift

```diff
@@ -22,15 +22,15 @@ struct CreateJobQueue: PostgresMigration {
             """
             CREATE TABLE IF NOT EXISTS _hb_pg_job_queue (
                 job_id uuid PRIMARY KEY,
-                createdAt timestamp with time zone
+                created_at timestamp with time zone
```

Comment:
Can we remove the variable renaming. If we are going to rename the variables we should do it in another PR